apps/portal-front/src/components/service/user_service.graphql.ts

```diff
@@ -180,6 +180,38 @@ export const UserServiceCreateMutation = graphql`
     }
   }
 `;
+export const UserServiceAddYourselfMutation = graphql`
+  mutation userServiceAddYourselfMutation(
+    $input: UserServiceAddYourselfInput!
+    $connections: [ID!]!
+  ) {
+    addYourselfInUserService(input: $input)
```

Comment:
selfRegisterInUserService(input: $input)
```suggestion
    addYourselfInUserService(input: $input)
```